src/dto/create-game-request.dto.ts

```diff
@@ -29,12 +31,16 @@ export class CreateGameRequestDto {
   @Max(30)
   totalRounds: MaxInt<30>;
 
-  @IsUrl()
-  @Matches(SPOTIFY_PLAYLIST_URL_REGEX)
-  @ValidateIf((o) => !o.genre || o.playlistUrl)
-  playlistUrl?: string;
+  @IsString()
+  @Length(22)
```

Comment:
why 22?
do they state that in their api?

Reply:
yes